stack-packs/index.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Pack as PackInterface } from './pack';
+
+declare const stackPacks: PackInterface[];
+
+export = stackPacks;
+export const Pack: PackInterface;
```

Comment:
do we need both the singular and the array version exported?

Reply:
If I want to use the non array version but seems like for-of fixes that.